lib/features/thread/presentation/widgets/email_tile_builder.dart

```diff
@@ -143,6 +146,17 @@ class EmailTileBuilder with BaseEmailItemTile {
     );
   }
 
+  Widget _buildCalenderIcon(PresentationEmail email) {
+    String mainString = email.emailHeader.toString();
```

Comment:
what is the `mainString`? You should use a more meaningful variable name

Reply:
I will delete it after getting the reel emailHeader from the PresentationEmail . i use it just to display the icons